CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
     the property is assigned to.
   - When finding all references it will find all places where the property
     is read with `var(--foo)` or `@apply --bar` syntax.
+- Supports code lenses to display information inline with your code.
+  - At a custom element declaration, displays how many places it's referenced
+    in HTML.
+  - At CSS Custom Property assignments, displays how many places it's read.
```

Comment:
a gif showing this would be awesome - but can be done later when preparing for the release 👌

Reply:
Yeah, many gifs are called for when this lands :)